lib/commands.coffee

```diff
@@ -51,12 +52,26 @@ class Commands
       atom.notifications.addError error.message, {detail: error.stack, dismissable: true}
 
   nextError: ->
-    if not @messages or (next = @messages.next()).done
-      next = (@messages = @linter.views.getMessages().values()).next()
-    return if next.done # There's no errors
-    message = next.value
-    return unless message.filePath
-    return unless message.range
+    if @index?
```

Comment:
True. I tried it with one linter, that was stable, but if there are tow and they report asynchronously it's going to be a problem as soon as one changes something.

The behavior before this PR was to always jump to the first reported error after an edit.